src/platform/datapath_raw_xdp.c

```diff
@@ -56,10 +60,17 @@ typedef struct _XDP_QUEUE {
     CXPLAT_LIST_ENTRY TxQueue;
 } XDP_QUEUE;
 
+typedef struct XDP_FLOW {
```

Comment:
I am little confused. What does a flow mean? Is it related to any design decisions we chose?

Reply:
So, now a datapath can have multiple interfaces and each interface can have multiple flows and each flow can have multiple queues?

Reply:
What is the purpose of this extra layer of abstration?

Reply:
Every new socket that creates will need to create a new queue + rule to receive only that traffic. See `XDP_QUIC_FLOW`.

Reply:
There's only one hit for XDP_QUIC_FLOW, which is in the struct definition of XDP_FLOW.

Every socket means literally every socket? Like for both client and listener? CxPlatXdpWorkerThread loops over each flow, which is going to be a problem when the number of flows is huge.

Reply:
For listeners, there's one socket per local UDP port used. For clients, yes, there is one per client connection.

Reply:
Yi and I discussed this a bit and decided on a slight change of direction. Please wait for the next commit.